spk/squidguard/src/app/squidGuard.cgi

```diff
@@ -83,8 +83,8 @@ $autoinaddr  = 2;			# 0|1|2;
 					# 2 TO AUTORESOLVE & REDIRECT TO FIRST NAME
 
 # You may wish to enter your company link and logo to be displayed on the page
-my $company = "SynoCommunity";
-my $companylogo = "synocommunity.png";
+my $company = "http://SynoCommunity.com";
+my $companylogo = "http://$ENV{SERVER_NAME}:5000/webman/3rdparty/squidguard/synocommunity.png";
```

Comment:
I would think this can be cleaned up a bit once you implement `DSM_UI_DIR`. I added a comment lower down where the same applies. Check out some other packages (e.g. a package like Debian-Chroot)